selenium/src/test/java/au/org/emii/portal/tests/step1/FacetOrganisationTest.java

```diff
@@ -84,7 +84,7 @@ private void verifyOrganisation(String org) {
 
         List<WebElement> results = webElementUtil.findElements(By.className("resultsTextBody"));
         for (WebElement result : results) {
-            WebElement facetResult = result.findElement(By.xpath("div[2]/span[2]"));
+            WebElement facetResult = result.findElement(By.xpath("//span[contains(@class, 'fa-institution')]/../following-sibling::span"));
```

Comment:
How about 

`div[3]/span[2]` 

instead of 

`//span[contains(@class, 'fa-institution')]/../following-sibling::span`?